globus_action_provider_tools/authentication.py

```diff
@@ -176,17 +213,37 @@ def get_authorizer_for_scope(
         token_expiration = dep_tkn_resp.get("expires_at_seconds")
         now = time()
```

Comment:
`now` is only used in one place. Would it be worthwhile to remove this variable and get the current `time()` in the one place where it's used?